internal/provider/resource_gitlab_service_emails_on_push_test.go

```diff
@@ -0,0 +1,146 @@
+//go:build acceptance
+// +build acceptance
+
+package provider
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+func TestAccGitlabServiceEmailsOnPush_basic(t *testing.T) {
+	testProject := testAccCreateProject(t)
+
+	var emailsOnPushService gitlab.EmailsOnPushService
+
+	var recipients1 = "mynumberonerecipient@example.com"
+	var recipients2 = "mynumbertworecipient@example.com"
+	var emailsOnPushResourceName = "gitlab_service_emails_on_push.this"
+
+	resource.ParallelTest(t, resource.TestCase{
+		ProviderFactories: providerFactories,
+		CheckDestroy:      testAccCheckGitlabServiceEmailsOnPushDestroy,
+		Steps: []resource.TestStep{
+			// Create an Emails on Push service
+			{
+				Config: testAccGitlabServiceEmailsOnPushConfig(testProject.ID, recipients1),
```

Comment:
We started inlining the configurations for better readability. Thus, could you please inline it?

See https://github.com/gitlabhq/terraform-provider-gitlab/issues/1008#:~:text=Prefer%20writing%20config%20in%2Dline%20to%20wrapping%20it%20in%20a%20function.